src/Compilers/Core/Portable/CommandLine/SarifV2ErrorLogger.cs

```diff
@@ -74,6 +80,29 @@ public override void LogDiagnostic(Diagnostic diagnostic, SuppressionInfo? suppr
                     _writer.Write("justification", justification);
                 }
 
+                string? suppressionType = null;
+                if (diagnostic.ProgrammaticSuppressionInfo is { } programmaticSuppressionInfo)
+                {
+                    var suppressionsStr = programmaticSuppressionInfo.Suppressions
+                        .OrderBy(idAndJustification => idAndJustification.Id)
+                        .Select(idAndJustification => $"Suppression Id: {idAndJustification.Id}, Suppression Justification: {idAndJustification.Justification}")
```

Comment:
>Suppression Id: {idAndJustification.Id}, Suppression Justification: {idAndJustification.Justification} [](http://example.com/codeflow?start=56&length=102)

Is this a JSON string? If not, are these names and values converted into JSON?

Reply:
No its just a ToString representation of the programmatic suppression info that is inserted in the the error log as a JSON value string. Do you feel there could be a better way to emit this data?

Reply:
> Do you feel there could be a better way to emit this data?

No, I just wanted to make sure I understood how it was represented, thanks.